docs/sql-programming-guide.md

```diff
@@ -1798,8 +1798,7 @@ DataFrame jdbcDF = sqlContext.load("jdbc", options)
 
 {% highlight python %}
 
-df = sqlContext.load(source="jdbc", url="jdbc:postgresql:dbserver", dbtable="schema.tablename")
-
+df = sqlContext.read.format('jdbc').options(url = 'jdbc:postgresql:dbserver', dbtable='schema.tablename').load()
```

Comment:
Oh hah of course it's Python, nevermind. I was thinking this was another Scala example after reading the ones above.